app/controllers/article_controller.rb

```diff
@@ -19,6 +19,25 @@ def show
 		@article = Article.find(params[:id])
 	end
 
+	def edit
+		@article = Article.find(params[:id])
+	end
+
+	def update
+		@article = Article.find(params[:id])
+
```

Comment:
do not use empty lines. 
use them to logically separate the code otherwise do not use them.